src/components/NavBar.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState, useContext } from "react";
+import { Route, Routes, Link } from "react-router-dom";
+import axios from "axios";
```

Comment:
As per previous comments, don't blindly import. Why does this component need the getAllArticles function? Why does it need Route?